css/style.css

```diff
@@ -44,42 +36,31 @@ div.autocomplete-deluxe-container input.autocomplete-deluxe-form {
   color: #003CC5;
 }
 
-.content-header .is-horizontal-enabled:not(.is-horizontal) .tabs__link.js-tabs-link {
-  color: #545560;
-}
-
 .field--name-field-content .layer-wrapper .field-multiple-table {
   margin-bottom: 16px;
 }
 
 .paragraphs-features__add-in-between__wrapper {
-  padding-bottom: 4px;
+  padding-bottom: 7px;
 }
 
-.field--type-social-media .form-type--checkbox .form-item__label {
+.field--type-social-media .form-type--checkbox .form-item__label{
   display: inline;
 }
 
-.dropbutton--single {
+.dropbutton.dropbutton--single {
   border: none;
 }
 
 .dropbutton--single .dropbutton__item--extrasmall .button {
-  line-height: 2.2rem;
-}
-
-.ui-dialog .paragraphs-add-dialog--categorized.ui-widget-content.ui-dialog-content {
-  max-height: 86vh!important;
+  line-height: 2.2rem!important;
+  margin: 0px!important;
```

Comment:
Please elaborate on why we need `!important` here

Reply:
for **save & edit button**  
1. line-height important not required removed it
2. **save & edit button**  without important it's takes margin from the other button class, tried to check for parent class but not working. that's way important is used here.